### ISSUE TITLE
pb-x sizes in helpers.scss are wrong direction

### ISSUE THREAD

**Issue by git-no** [mdbootstrap/bootstrap-material-design#48]
padding bottom should be a padding at the bottom, but in helpers.scss it sets a padding top! Wrong!
Here how it is, it should be fixed!
```
.pb-1 {
    padding-top: 1rem !important;
}

.pb-2 {
    padding-top: 2rem !important;
}

.pb-3 {
    padding-top: 3rem !important;
}
````

**Comment by finkbg** [comment]
Yeah, that's true. I thought I was crazy.

**Comment by mattonit** [comment]
Fixed with 8b5e346 commit
